spec/features/tasks_spec.rb

```diff
@@ -33,9 +33,9 @@
       click_link 'Create Task'
       expect(page).to have_selector('#task_name', visible: true)
       fill_in 'task_name', with: 'Task I Need To Do'
-      chosen_select('Tomorrow', from: 'task_bucket')
-      chosen_select('Myself', from: 'task_assigned_to')
-      chosen_select('Call', from: 'task_category')
+      select 'Tomorrow', from: 'task_bucket'
+      select 'Myself', from: 'task_assigned_to'
+      select 'Call', from: 'task_category'
```

Comment:
Please also remove the test helper method `chosen_select` itself, since chosen is no longer used anywhere